target/linux/kirkwood/base-files/etc/board.d/03_gpio_switches

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board_config_update
+
+board=$(board_name)
+
+case "$board" in
+checkpoint,l-50)
+	ucidef_add_gpio_switch "mpcie-rst" "mPCIE Card reset" "502" "1"
+	ucidef_add_gpio_switch "exp-card-rst" "Express Card reset" "497" "1"
+;;
```

Comment:
Add a indent as in the other files.

Reply:
It was taken from:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/mt7620/base-files/etc/board.d/03_gpio_switches

@adrianschmutzler 
Should be added indent?

Reply:
I mean just for `;;`.

Reply:
Ah, sorry. To early.

Reply:
as mans0n says, just the ;; (didn't understand first as well :-) )